src/docs/implementations/dynatrace.adoc

```diff
@@ -13,19 +13,21 @@ include::install.adoc[]
 For setting up new integrations with Dynatrace, it is recommended to use the latest version of the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/[Dynatrace Metrics API] (v2).
 Dynatrace provides different ways of setting up integrations:
 
-=== Using a Dynatrace OneAgent installed on the host (preferred) [[bookmark-oneagent-example]]
+=== Using Dynatrace auto-configuration (preferred) [[bookmark-auto-configuration]]
+
+Dynatrace auto-configuration is available for hosts that are monitored by a OneAgent or by the Dynatrace Operator for Kubernetes.
 
 If a Dynatrace OneAgent is installed on the host running Micrometer, metrics can be exported directly using the OneAgent without having to specify an endpoint URI or API token.
-Use the following code in your project to export Micrometer metrics to the https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/[local OneAgent endpoint]:
+If running in Kubernetes with the Dynatrace operator installed, the registry will pick up your endpoint URI and API token from the operator instead.
+In this case there is no need to configure anything, you can use the following code in your project to export Micrometer metrics to Dynatrace:
 
 [source,java]
 ----
 DynatraceConfig dynatraceConfig = new DynatraceConfig() {
     @Override
     @Nullable
     public String get(String k) {
-        // This method can be used for retrieving arbitrary config items,
-        // null means accepting the defaults defined in DynatraceConfig
+        // This method for retrieving arbitrary config items introduced by the interface is currently not used in DynatraceConfig.
```

Comment:
A bit of a nit-pick: the `get(String)` method is used by all the other config methods' default implementation. This is what allows simple lambda implementations of our Config classes that can retrieve config from `System` properties or env vars. e.g.:

```java
DynatraceConfig system = System::getProperty;
DynatraceConfig env = System::getenv;
```

Reply:
Would something like:
```
// This method of the interface is used by the other configuration methods' default implementations.
```
work? There is no way of omitting this entirely, right? It seems a bit clunky to have to specify this every time.

Reply:
Some other Config provide a static `DEFAULT` in the Config interface for convenience like: `DynatraceConfig DEFAULT = key -> null;` We could add this to `DynatraceConfig`.

Reply:
In addition to that, users can use the lambda instead of an anonymous class, which is verbose, if they just want the defaults.

Reply:
That is a great suggestion, thanks!